lua/lscs/autorun/lscs_player.lua

```diff
@@ -64,6 +64,7 @@ if SERVER then
 	end
 
 	function meta:lscsGetForceAllowed()
+		if self:GetNWBool( "lscs_force_allow_useage" ) == false then return false end
```

Comment:
i rather check the variables that are responsible for it. This change isnt needed